lighthouse-core/audits/seo/plugins.js

```diff
@@ -73,7 +73,7 @@ function isPluginURL(url) {
     if (parts.length < 2) {
       return false;
     }
-    const part = /** @type {string} */(parts.pop());
+    const part = parts[parts.length - 1];
```

Comment:
😆  this is a funny one :)

Reply:
I had the same thought :)

Reply:
I didn't say all of these were Platonic monuments to type checking, just that the type casts weren't needed :D